### PR TITLE
Support tuple/list as axis argument in BatchNormalizatio` layer

### PR DESCRIPTION
**Pull request by vcarpani**
### Summary
This PR add support tuple/list as `axis` argument in `BatchNormalization` layer, as implemented in `tf.keras.layers.BatchNormalization`
### Related Issues
This PR solves issue https://github.com/keras-team/keras/issues/12126
### PR Overview

- [x] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [x] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by gabrieldemarmiesse** [comment]
Can you please add unit tests for this new feature?

**Comment by vcarpani** [comment]
Yes, I will write them
